assets/src/components/UserSettingSnackbar.js

```diff
@@ -2,6 +2,12 @@ import React, { useState, useEffect } from 'react'
 import Snackbar from '@material-ui/core/Snackbar'
 import IconButton from '@material-ui/core/IconButton'
 import CloseIcon from '@material-ui/icons/Close'
+import Fade from '@material-ui/core/Fade';
```

Comment:
You should remove this since you aren't using the `Fade` component.

Reply:
Sure